src/components/services/AirshipInstance.tsx

```diff
@@ -90,6 +91,20 @@ export function showWarning(error: unknown, options: ShowErrorWarningOptions = {
   Airship.show(bridge => <AlertDropdown bridge={bridge} message={translatedError} warning />).catch(err => console.error(err))
 }
 
+/**
+ * Shows a persistent warning to the user. Can only be dismissed on pressing the
+ * close button of the drop-down or tapping the airship itself.
+ */
+export function showPersistentWarning(error: unknown, options: ShowErrorWarningOptions = {}): void {
+  const { trackError: doTrackError = true, tag, onPress } = options
+  const translatedError = tag ? `Tag: ${tag}. ` + translateError(error) : translateError(error)
+  if (doTrackError) {
+    trackError(`showWarning: ${translatedError}`)
+  }
+  console.log(yellowText('Showing warning drop-down alert: ' + makeErrorLog(error)))
+  Airship.show(bridge => <AlertDropdown bridge={bridge} message={translatedError} onPress={onPress} persistent warning />).catch(err => console.error(err))
+}
```

Comment:
Please delete this function. Whoever was going to use it can just do this:

```ts
await Airship.show(bridge => <AlertDropdown
   bridge={bridge}
   message="your text here"
   onPress={onPress}
   persistent
   warning
   />)
```

The `showError` and `showDevError` functions are designed for error-reporting, of which showing a drop-down is just a optional sub-feature. We don't need all that ceremony to show a drop-down. We can treat it like just another modal.